armi/bookkeeping/db/tests/test_database3.py

```diff
@@ -194,6 +197,14 @@ def test_computeParents(self):
             database3.Layout.computeAncestors(serialNums, numChildren, 3), expected_3
         )
 
+    def test_load(self) -> None:
+        self.makeShuffleHistory()
+        with self.assertRaises(KeyError):
+            r = self.db.load(0, 0)
+
+        del self.db.h5db["c00n00/Reactor/missingParam"]
+        r = self.db.load(0, 0)
```

Comment:
Why delete and not just test passing the `allowMissing` argument as True?

Reply:
To make sure that the extra param is _why_ the KeyError is getting raised

Reply:
Right, but what about a test that allows the loading _not_ to fail when the parameter doesn't exist?